admin/class-admin.php

```diff
@@ -71,6 +71,8 @@ public function __construct() {
 
 		add_action( 'admin_init', array( $this, 'map_manage_options_cap' ) );
 
+		add_action( 'init', array( $this, 'check_php_version' ) );
```

Comment:
As this functionality only needs to run in the `admin` area, this should be `admin_init`